nRF51GattClient.h

```diff
@@ -116,14 +116,11 @@ class nRF51GattClient : public GattClient
     }
 
     virtual ble_error_t write(GattClient::WriteOp_t cmd, Gap::Handle_t connHandle, GattAttribute::Handle_t attributeHandle, size_t length, const uint8_t *value) const {
-        ble_gattc_write_params_t writeParams = {
-            .write_op = cmd,
-            // .flags  = 0,
-            .handle   = attributeHandle,
-            .offset   = 0,
-            .len      = length,
-            .p_value  = const_cast<uint8_t *>(value),
-        };
+        ble_gattc_write_params_t writeParams = { };
```

Comment:
This was annoying.

G++ does **not** support C99/C11-style designated initializers, because ISO-C++ does not have designated initializers. G++ sort-of supports them with the `gnu++11` flavour of C++, but with a different syntax, unfortunately.

Very annoying, but trivial to fix.